### PR TITLE
Format api output

### PR DESCRIPTION
**Pull request by dustints**

### CODE REVIEW

File: app/lib/actions/foreman_gutterball/content_reports/report_formatter.rb [hunk 1]
```diff
@@ -0,0 +1,138 @@
+module Actions
```
Comment (komidore64) [inline app/lib/actions/foreman_gutterball/content_reports/report_formatter.rb]: i'm not sure `app/lib/actions` is the best place for the `ReportFormatter`. `app/lib/actions/blah` is where dynflow actions go. I would argue maybe `app/lib/foreman_gutterball/formatters` or `lib/foreman_gutterball` would be a better place.

File: test/unit/report_formatter_test.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        "name" : "test-consumer-BEcQrEdg",
+        "owner" : {
+          "displayName" : "ACME Corporation",
+          "key" : "acme_corporation"
+        },
+        "lastCheckin" : "2015-02-09T13:27:37.809+0000",
+        "uuid" : "0668eca4-2efe-4965-a2ea-610027164c4e"
+      }
+    }
+    EOF
```
Comment (komidore64) [inline test/unit/report_formatter_test.rb]: is this supposed to be the full output of a gutterball report?

Comment (dustints) [inline test/unit/report_formatter_test.rb]: @komidore64 yah let me update with a full response.

File: app/services/foreman_gutterball/gutterball_service.rb [hunk 3]
```diff
@@ -20,20 +22,67 @@ def self.logger
...
+      end
     end
 
-    def run_reports(report_key, query_params = nil)
-      report_path = "#{prefix}reports/#{report_key}/run?#{query_params.to_query}"
-      get_reports(report_path, default_headers)
+    def report(report_key, query_params)
+      format_query(query_params)
+      path = self.class.join_path(prefix, 'reports', report_key, 'run', hash_to_query(query_params))
+      # might need a SAX parser after looking at all that data
```
Comment (komidore64) [inline app/services/foreman_gutterball/gutterball_service.rb]: let's remove this comment

Comment (komidore64) [inline app/services/foreman_gutterball/gutterball_service.rb]: removed.

**Comment by komidore64** [comment]
testing an RPM build to make sure rpmbuild doesn't scream about anything unexpected.

**Comment by komidore64** [comment]
looks good

```
$ tito build --test --rpm --scl ruby193 --builder mock --arg mock=katello-scl
WARNING: unable to lookup latest package tag, building untagged test project
WARNING: rel-eng/packages/rubygem-foreman_gutterball doesn't exist in git, using current directory
WARNING: unable to lookup latest package tag, building untagged test project
WARNING: rel-eng/packages/rubygem-foreman_gutterball doesn't exist in git, using current directory
Building package [rubygem-foreman_gutterball-0.0.1-1]
Creating rpms for rubygem-foreman_gutterball-git-0.e6b1bc1 in mock: katello-scl
Wrote: /tmp/tito/ruby193-rubygem-foreman_gutterball-0.0.1-1.git.0.e6b1bc1.el6.src.rpm

Using srpm: /tmp/tito/ruby193-rubygem-foreman_gutterball-0.0.1-1.git.0.e6b1bc1.el6.src.rpm
Initializing mock...
Installing deps in mock...
Building RPMs in mock...

Wrote:
  /tmp/tito/ruby193-rubygem-foreman_gutterball-0.0.1-1.git.0.e6b1bc1.el6.noarch.rpm
```

ACK

**Pull request merged**